### PR TITLE
Create file structure

### LINKED ISSUE #5: File Structure
## File Structure

- #### client/
    - public/
    - src/
      -  app/
          - index.js
          - app.css
      - assets/
         - icons
         - images
      - components/
           - index.js
      - utils/
        - index.js
      - pages/
        - index.js
   - .gitignore
   - .eslintrc.json
   - package-lock.json
   - package.json
   
- #### server/
     - controllers/
       - middlewares/
          -  auth/
             -  index.js 
        - errorHandle/
           - index.js
      - routes/
         - index.js
     - utils/
       - index.js
     -  validation/
         - index.js
    - database/
        - build.sql
        - fakeData.sql
        - build.js
        - connection.js	
            
    - queries/
        - index.js
    - app.js
    - index.js
    - .env
- test/
- .gitignore
- .eslintrc.json
- package-lock.json
- package.json

### PR DESCRIPTION
**Pull request by Jehad91**
Relates #5
Close #5

### CODE REVIEW

**Review by MostafaQanoo — APPROVED** [review]

**Review by MohammadAlHabil — APPROVED** [review]

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
delete React.StrictMode and reportWebVitals from index file in client folder
```
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);

// If you want to start measuring performance in your app, pass a function
// to log results (for example: reportWebVitals(console.log))
// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
reportWebVitals();
```

**Review by IsraaHamdi — DISMISSED** [review]

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.1.1",
+    "@testing-library/user-event": "^13.5.0",
```
Comment (Mu7ammadAbed) [inline client/package.json:8]:
```suggestion
```

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```
Comment (Mu7ammadAbed) [inline client/package.json:12]:
```suggestion
```

File: client/src/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
```
Comment (Mu7ammadAbed) [inline client/src/index.js:3]: You need to export the `App` in order to import it

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]
You need to change the type of files in the `server` to `.ts`

**Review by MostafaQanoo — APPROVED** [review]

**Review by IsraaHamdi — APPROVED** [review]

**Review by MohammadAlHabil — APPROVED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**